accname/name/comp_labelledby_hidden_nodes.html

```diff
@@ -0,0 +1,266 @@
+    <span id="b61" style="visibility: hidden;">
+        foo
+        <span id="b62">bar</span>
+        <span id="b63">baz</span>
+    </span>
+
+<h2>Testing with <code>visibility:collapse</code></h2>
+
+    <button aria-labelledby="c11" data-expectedlabel="foo bar" data-testname="button with aria-labelledby using visibility:collapse span (with nested span)" class="ex"></button>>x</button>
+    <span id="c11" style="visibility: collapse;">
+        foo
+        <span id="c12" style="visibility: collapse;">bar</span>
+    </span>
+
+    <button aria-labelledby="c21" data-expectedlabel="foo bar baz" data-testname="button with aria-labelledby using visibility:collapse hidden span (with nested visibility:collapse spans, depth 2)" class="ex"></button>>x</button>
+    <span id="c21" style="visibility: collapse;">
+        foo
+        <span id="c22" style="visibility: collapse;">
+            bar
+        <span id="c23" style="visibility: collapse;">baz</span>
+        </span>
+    </span>
+
+    <button aria-labelledby="c31" data-expectedlabel="foo" data-testname="button with aria-labelledby using span without visibility:collapse (with nested visibility:collapse spans, depth 2)" class="ex"></button>>x</button>
+    <span id="c31">
+        foo
+        <span id="c32" style="visibility: collapse;">
+            bar
+        <span id="c33" style="visibility: collapse;">baz</span>
+        </span>
+    </span>
+
+    <button aria-labelledby="c41" data-expectedlabel="foo bar baz" data-testname="button with aria-labelledby using visibility:collapse hidden span (with nested visibility:collapse sibling spans)" class="ex"></button>>x</button>
+    <span id="c41" style="visibility: collapse;">
+        foo
+        <span id="c42" style="visibility: hidden;">bar</span>
+        <span id="c43" style="visibility: hidden;">baz</span>
+    </span>
+
+    <button aria-labelledby="c51" data-expectedlabel="foo" data-testname="button with aria-labelledby using span without visibility:collapse (with nested visibility:collapse sibling spans)" class="ex"></button>>x</button>
+    <span id="c51">
+        foo
+        <span id="c52" style="visibility: collapse;">bar</span>
+        <span id="c53" style="visibility: collapse;">baz</span>
+    </span>
+
+    <button aria-labelledby="c61" data-expectedlabel="foo bar baz" data-testname="button with aria-labelledby using span with visibility:collapse (with nested visible sibling spans)" class="ex"></button>>x</button>
+    <span id="c61" style="visibility: collapse;">
+        foo
+        <span id="c62">bar</span>
+        <span id="c63">baz</span>
+    </span>
+
+<h2>Testing with <code>aria-hidden</code></h2>
+
+    <button aria-labelledby="d11" data-expectedlabel="foo bar" data-testname="button with aria-labelledby using aria-hidden span (with nested aria-hidden span)" class="ex"></button>>x</button>
+    <span id="d11" aria-hidden="true">
+        foo
+        <span id="d12" aria-hidden="true">bar</span>
```

Comment:
I think the only way aria-hidden inside aria-hidden is not redundant is:
1) if you're testing aria-hidden=false (which I think the spec recommends against)
2) if you're testing when the label is no longer ignored on hidden nodes while they are focused

So I'd remove any extra `aria-hidden=true` attrs here and below.